src/frontends/tensorflow_lite/src/op/gelu.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2024 Intel Corporation
```

Comment:
please remove this file. It looks not needed. That translator was added to `tensorflow_common`

Reply:
This file is needed to take care of the TFL attribute. It should stay

Reply:
@jane-intel, you are correct. Attribution helper is indeed needed for TF Lite, I forgot. Thanks for catch-up:)